src/service/api/categories.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const {Router} = require(`express`);
+const {HttpCode} = require(`../constants`);
+
+const route = new Router();
+
+module.exports = (app, service) => {
```

Comment:
```suggestion

module.exports = (app, service) => {

const route = new Router();

```
Перемести чтобы во всех роутах вызов был внутри функций. Если так не сделать, то дальше могут быть странные ситуации, которые сложно дебажить